tests/e2e_operations.py

```diff
@@ -354,6 +355,41 @@ def run_tls_san_checks(args):
         assert len(sans) == 1, "Expected exactly one SAN"
         assert sans[0].value == ipaddress.ip_address(dummy_public_rpc_host)
 
+    # This is relatively direct test to make sure the config timeout feature
+    # works as intended. It is difficult to do with the existing framework
+    # as is because of the indirections and the fact that start() is a
+    # synchronous call.
+    start_node_path = network.nodes[0].remote.remote.root
+    # Remove ledger and pid file to allow a restart
+    shutil.rmtree(os.path.join(start_node_path, "0.ledger"))
+    os.remove(os.path.join(start_node_path, "node.pid"))
+    os.remove(os.path.join(start_node_path, "service_cert.pem"))
+    # Save configuration
+    shutil.copy(
+        os.path.join(start_node_path, "0.config.json"),
+        os.path.join(start_node_path, "0.config.json.bak"),
+    )
+    # Replace it with a prefix
+    with open(os.path.join(start_node_path, "0.config.json"), "w") as f:
+        f.write("{")
+    LOG.info("Attempt to start node with a partial config under {start_node_path}")
```

Comment:
```suggestion
    LOG.info(f"Attempt to start node with a partial config under {start_node_path}")
```

Can we test that the node also spins (happily) if there's _no_ file for a few seconds??

Reply:
Sure